packages/teleport/src/services/user/makeAcl.ts

```diff
@@ -32,9 +33,11 @@ export default function makeAcl(json): Acl {
   const accessRequests = json.accessRequests || defaultAccess;
   const billing = json.billing || defaultAccess;
   const dbServers = json.dbServers || defaultAccess;
+  const desktopServers = json.desktopServers || defaultAccess;
 
   return {
     logins,
+    windowsLogins,
```

Comment:
windows or desktop logins?

Reply:
In the RFD Andrew was pretty intentional about dedicated "Windows Logins" instead of using the existing field we have for *nix logins, so let's keep them separate.